README.md

```diff
@@ -3,14 +3,22 @@
 
 ![Screenshot of PyCob Quickstart App](https://cdn.pycob.com/quickstart_screenshot.png)
 
-## Create web apps using just Python
+We created PyCob to let you quickly create functional apps using only python. Read more about the project goals here
+
 
+## Create web apps using just Python
+The app from the above screenshot can be started/created/is possible with just the following steps
 ### Install PyCob
 ```bash
+
```

Comment:
There is a makefile in the quickstart repo.  I would recommend against this.  Python devs don't expect a makefile,  and the instructions are simple enough that listing them out manually here is enough